WhisperAPI/WhisperAPI/Settings/ApplicationSettings.cs

```diff
@@ -1,7 +1,13 @@
-﻿namespace WhisperAPI.Settings
+﻿using System.Collections.Generic;
+
+namespace WhisperAPI.Settings
 {
     public class ApplicationSettings
     {
         public string ApiKey { get; set; }
+
+        public string NlpApiBaseAddress { get; set; }
+
+        public List<string> Intents { get; set; }
```

Comment:
Same as in "SuggestionsService" : Not clear enough what List<string> Intents is used for (are they relevant intent or intent that we want to discard?)

Reply:
Done